doc/api/cli.md

```diff
@@ -138,6 +138,10 @@ Force FIPS-compliant crypto on startup. (Cannot be disabled from script code.)
 
 Specify ICU data load path. (overrides `NODE_ICU_DATA`)
 
+### `--preserve-symlinks`
+
+Instructs the module loader to preserve symbolic links when resolving and
+caching modules.
```

Comment:
Is a commandline option the right way to solve the #3402 issue? What about node scripts executed from npm's package.json for example? Should npm (or other scripts invoking node as the executable - written in JavaScript, shell and what not) have a special facility to supply this flag to its node children magically? For example, suppose have [the folllowing stanza in the package.json](https://github.com/sass/node-sass/blob/6027012135944fbec30e6ebc2252f223199961cf/package.json#L29-L37):

````
"scripts": {
    "coverage": "node scripts/coverage.js",
    "install": "node scripts/install.js",
    "postinstall": "node scripts/build.js",
    "pretest": "node_modules/.bin/jshint bin lib scripts test",
    "test": "node_modules/.bin/mocha test",
}
````

The flag will not be automatically passed to `mocha` whenever we run `npm test`.